moveit_ros/planning_interface/moveit_cpp/include/moveit/moveit_cpp/moveit_cpp.h

```diff
@@ -130,7 +131,7 @@ class MoveItCpp
   robot_model::RobotModelConstPtr getRobotModel() const;
 
   /** \brief Get the ROS node handle of this instance operates on */
-  const ros::NodeHandle& getNodeHandle() const;
+  const rclcpp::Node::SharedPtr& getNode() const;
```

Comment:
ahh. Now I understand. If you need to get the node out than you have to have the full shared ptr.

Reply:
Yep ;)